apps/web/src/modules/auction/components/CurrentAuction/PlaceBid.tsx

```diff
@@ -182,16 +183,38 @@ export const PlaceBid = ({
               </Box>
             </Box>
           </form>
-
-          <ContractButton
-            className={auctionActionButtonVariants['bid']}
-            handleClick={handleCreateBid}
-            disabled={address && isValidChain ? !isValidBid : false}
-            mt={{ '@initial': 'x2', '@768': 'x0' }}
-          >
-            Place bid
-          </ContractButton>
-        </Fragment>
+          <Flex mt="x2">
```

Comment:
can we try and keep this section on the same line as the input like in the mock @isaaczaak shared